src/main/scala/diplomacy/Nodes.scala

```diff
@@ -1235,14 +1235,14 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
       data   = bundleIn(i))
   }
 
-  private var bundlesSafeNow = false
+  private[diplomacy] var instantiated = false
 
   /** Gather Bundle and edge parameters of outward ports.
     *
     * Accessors to the result of negotiation to be used within [[LazyModuleImp]] Code.
```

Comment:
Maybe update this to `should only be used within [[LazyModuleImp]] code or after its instantiation has completed.`?